packages/desktop-electron/package.json

```diff
@@ -19,7 +21,9 @@
       "!**/*.js.map",
       "!node_modules/@jlongster/sql.js",
       "!node_modules/absurd-sql",
-      "!node_modules/better-sqlite3/{benchmark,src,bin,docs,deps,build/Release/obj,build/Release/sqlite3.a,build/Release/test_extension.node}"
+      "!node_modules/better-sqlite3/{benchmark,src,bin,docs,deps,build/Release/obj,build/Release/sqlite3.a,build/Release/test_extension.node}",
+      "build",
```

Comment:
Specifying the built files are required for the bundle - used by electron-builder to package them